modules/HTTP-Authentication/README.md

```diff
@@ -68,6 +68,8 @@ There are a ton of specs you need to build so make sure you break each problem d
         or password"
       - with a matching email and password
         - you should be logged in and redirect to the home page
+    - and you visit `/logout`
+      - you are redirected to the home page
```

Comment:
I think this one should be `you should be logged out and redirect to the home page`